src/controllers/controller.bubble.js

```diff
@@ -80,6 +80,7 @@ module.exports = function(Chart) {
 					y: reset ? yScale.getBasePixel() : yScale.getPixelForValue(data, index, dsIndex),
 					// Appearance
 					radius: reset ? 0 : custom.radius ? custom.radius : me.getRadius(data),
+					pointStyle: reset ? pointElementOptions.pointStyle : custom.pointStyle ? custom.pointStyle : helpers.getValueAtIndexOrDefault(dataset.pointStyle, index, pointElementOptions.pointStyle),
```

Comment:
Does this use all the same point styles as the line chart? If so, there needs to be tests and documentation for this change